recipe/meta.yaml

```diff
@@ -27,9 +20,8 @@ requirements:
     - cmake
     - make        # [unix]
     - pkg-config  # [unix]
-    - python
   host:
-    - msinttypes  # [win and vc==9]
+    - python
```

Comment:
Why is `python` a `host` requirement? This only used as a build tool still, right?

Reply:
I just moved it because I thought it was using the location of the python executable to find the python libs and things like that. 
Should I return it to ``build`` then?

Reply:
I think it is just using it to run some tests.

Reply:
So yeah moving it back to `build` would be good.